bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,105 @@
+{% add_lang_files "firefox/products/lockwise" %}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_favicon %}{{ static('img/firefox/lockwise/favicon.ico') }}{% endblock %}
+
+{% block page_title %}{{_('Firefox Lockwise — password manager —  take your passwords everywhere')}}{% endblock %}
+
+{% block page_desc %}{{ _('Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('lockwise') }}
+{% endblock %}
+
+{% block body_id %}firefox-lockwise{% endblock %}
+{% block body_class %}firefox-lockwise{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <section class="mzp-c-hero mzp-has-image">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <img src="{{ static('img/firefox/lockwise/lockwise-horizontal.svg') }}" class="lockwise-logo" />
```

Comment:
@rraue pointed out in Slack that this should contain some text for SEO since it's the product name. I'd suggest trying a heading here, with the `image-replaced` mixin (which should replace `<h1>Firefox Lockwise</h1>` with the wordmark image).

Reply:
I'm not sure how to make this work with hero macro. Should I unwind and not use the hero macro?

Reply:
I linked to an example for how you can do this below in the comments, but there should be no need to rewind the macro. Happy to discuss in person if it helps?

Reply:
I realize I was thinking to firmly inside the box... I can just not add the Lockwise image header to the hero 🤦‍♀